drivers/SmartThings/matter-lock/profiles/lock-unlatch.yml

```diff
@@ -0,0 +1,23 @@
+name: lock-unlatch
+components:
+- id: main
+  capabilities:
+  - id: lock
+    version: 1
+    config:
+      values:
+      - key: "lock.value"
+        enabledValues:
```

Comment:
I think `unlatched` needs to be added to `enabledValues` in all these new *-unlatch profiles

Reply:
Yes, I will fix it.